### PR TITLE
Remove validator subcommands.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Remove validator subcommands.

### CODE REVIEW

File: docs/HowTo/Get-Started/Connect-To-Testnet.md [hunk 1]
```diff
@@ -83,76 +78,30 @@ requires 32 Goerli ETH per validator.
...
-!!! tip
+Use the [Medalla Launchpad] to guide you through a step-by-step process to generate your keys and
+send the deposits.
 
-    For small numbers of validators, we recommend using the [Medalla Launchpad]
-    to generate your keys and send the deposits. For more than around 10
-    validators, the workflow below is more convenient.
+!!! note
+    Remember the passwords that you used to create the validator keys, because you need it to [create
+    the validator password files](#create-a-password-file-for-each-validator-key).
```
Comment (NicolasMassart) [inline docs/HowTo/Get-Started/Connect-To-Testnet.md:88]: The link is not going to work if split on two lines

**Review by cemozerr — APPROVED** [review]
LGTM.

File: docs/HowTo/Get-Started/Connect-To-Testnet.md [hunk 2]
```diff
@@ -83,76 +78,30 @@ requires 32 Goerli ETH per validator.
...
 
-    For the Zinken testnet we recommend you use the [Zinken Launchpad] only.
```
Comment (mbaxter) [inline docs/HowTo/Get-Started/Connect-To-Testnet.md:99]: Do we want to keep a note about Zinken somewhere?  We wouldn't want people to accidentally send deposits to medalla when they meant to send to zinken.

Comment (bgravenorst) [reply docs/HowTo/Get-Started/Connect-To-Testnet.md:99]: Done

**Pull request merged**